components/eamxx/src/physics/shoc/tests/infra/shoc_data.cpp

```diff
@@ -112,31 +107,12 @@ FortranDataIterator::getfield (Int i) const {
   return fields_[i];
 }
 
-void shoc_init(Int nlev, bool use_fortran, bool force_reinit) {
-  static bool is_init = false;
-  if (!is_init || force_reinit) {
-    using Scalar = Real;
-    using C = scream::physics::Constants<Scalar>;
-
-    shoc_init_c((int)nlev, C::gravit, C::Rair, C::RH2O, C::Cpair, C::ZVIR,
-                C::LatVap, C::LatIce, C::Karman, C::P0);
-    is_init = true;
-  }
-  shoc_use_cxx_c(!use_fortran);
-}
-
 int test_FortranData () {
```

Comment:
Does this mean that even thought we've dropped F90 support we still need to use some data in the Fortran structure somewhere?  Or just test for it?

Reply:
Perhaps it can be done as a follow up PR, but I would also argue in favor of pruning any reference to Fortran, like `FortranData` and `FortranDataIterator`. If they are still needed, we should rename into something like `HostData` or `BaselineData`, or just `TestData`...